File: codigos/1 - estrutura sequencial/1011 - Esfera/Main.cc
// Online C compiler to run C program online
#include <stdio.h>
#include <math.h>
#define PI 3.14159
#define VOLUME(raio) (4/3.0 * PI * pow(raio, 3))

int main() {
    int raio; 
    scanf("%d", &raio);
    printf("VOLUME = %.3lf\n", VOLUME(raio));
    return 0;
}

File: codigos/1 - estrutura sequencial/1011 - Esfera/Main.clj
(def PI 3.14159)

(defn get-volume [raio]
    (* (/ 4 3.0) PI (Math/pow raio 3)))

(defn main []
    (let [raio (read)
          volume (get-volume raio)]
        (println "VOLUME =" (format "%.3f" volume))))

(main)